config/routes.rb

```diff
@@ -1,5 +1,16 @@
 Rails.application.routes.draw do
   devise_for :users
-  root to: 'pages#home'
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  root to: 'pages#home', as: :home
+
+  resources :hangouts do
+    resources :hangout_chats, only: :show
+    resources :hangout_messages, only: [:create, :edit, :update, :destroy]
```

Comment:
Does this need to be nested in hangout_chats?

Reply:
I feel like it does! Like:

 resources :hangouts do
    resources :hangout_chats, only: :show do
      resources :hangout_messages, only: [:create, :edit, :update, :destroy]
  ....

Right?